Scripts/Utilities/UnityBrandColorScheme.cs

```diff
@@ -136,7 +145,19 @@ public static class UnityBrandColorScheme
 		/// A unique Unity brand color gradient that can be set manually
 		/// UI elements (or otherwise) can fetch this common gradient, for a uniform appearance across various elements
 		/// </summary>
-		public static GradientPair sessionGradient { get; set; }
+		public static GradientPair sessionGradient
+		{
+			get { return s_SessionGradient; }
+			set
+			{
+				s_SessionGradient = value;
+
+				// Prevent the fetching of the session gradient from any of the curated collections
```

Comment:
That's apparent from the method naming. How come though? Can you change your comment to include the why?